lib/config/index.js

```diff
@@ -136,4 +138,30 @@ function checkLimitsConfig (cfg) {
   }
 }
 
+/** Check the format of the error-codes config files
+ *
+ */
+
+function checkErrorCodes() {
+
+  for (const domain of Object.keys(errorCodes)) {
+    let name = errorCodes[domain];
+    if (!name.hasOwnProperty('subdomains') || !name.hasOwnProperty('code')) {
```

Comment:
you also have to do type-checking

and as for the error code, additional to the type that must be an integer, you also have to make sure it's within the allowed range of codes

same goes for the checks on the subdomains and errors contents